lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -2374,6 +2468,10 @@ def _parse_comparisons(self):
             entrypoint='list',
             etc_hosts='set',
             ulimits='set(dict)',
+            device_read_bps='list(dict)',
```

Comment:
I think it should be `set(dict)`, since the order of the entries doesn't matter. Or does it?

(Same for the three below.)

Reply:
Yes. order of the entries does not matter here. I will update `list(dict)` to `set(dict)`.